vector/src/main/java/im/vector/app/features/home/room/list/RoomListSectionBuilderSpace.kt

```diff
@@ -65,7 +64,7 @@ class RoomListSectionBuilderSpace(
         private val onUpdatable: (UpdatableLivePageResult) -> Unit,
         private val suggestedRoomJoiningState: LiveData<Map<String, Async<Unit>>>,
         private val onlyOrphansInHome: Boolean = false
-) : RoomListSectionBuilder {
```

Comment:
I would maybe rename this class to `RoomListSectionBuilder` since it's now the only one